MapboxNavigation/RouteMapViewController.swift

```diff
@@ -469,11 +471,22 @@ class RouteMapViewController: UIViewController {
     @objc func resetFrameRate(_ sender: UIGestureRecognizer) {
         mapView.preferredFramesPerSecond = NavigationMapView.FrameIntervalOptions.defaultFramesPerSecond
     }
-
-    var contentInsets: UIEdgeInsets {
-        let top = instructionsBannerContentView.bounds.height
-        let bottom = bottomBannerContainerView.bounds.height
-        return UIEdgeInsets(top: top, left: 0, bottom: bottom, right: 0)
+    
+    func contentInset(forOverviewing overviewing: Bool) -> UIEdgeInsets {
+        let instructionBannerHeight = instructionsBannerContentView.bounds.height
+        let bottomBannerHeight = bottomBannerContainerView.bounds.height
+        
+        var insets = UIEdgeInsets(top: instructionBannerHeight, left: 0,
+                                  bottom: bottomBannerHeight, right: 0)
+        
+        if overviewing {
+            insets += NavigationMapView.courseViewMinimumInsets
+            
+            let routeLineWidths = MBRouteLineWidthByZoomLevel.compactMap { $0.value.constantValue as? Int }
+            insets += UIEdgeInsets(floatLiteral: Double(routeLineWidths.max() ?? 0))
```

Comment:
👍 As noted in https://github.com/mapbox/mapbox-navigation-ios/pull/2047#discussion_r267557733, this is rather conservative considering that only one zoom level is active at a given time. But we don‘t have ready access to the evaluated expression (unlike if we were using GL JS), and this method also gets called before calculating a camera rather than after, so choosing the right width isn’t worth the hassle. In practice, it would only be a few extra points on each side at most.